chore(amplifytools): amplifytools deprecation notice (#1066)
aws-amplify/amplify-ios: AmplifyTools/README.md

## ⚠️ Deprecation notice

> The build phase script provided by the `Amplify/Tools` pod is not longer recommended. It's *currently deprecated* and it **will be completely removed in a future release**.

Reviewer: ```suggestion
The build phase script provided by the `Amplify/Tools` pod is no longer recommended. It's *currently deprecated* and it **will be completely removed in a future release**.
```

> The functionality provided by the `amplify-tools.sh` is currently integrated with the Amplify CLI and iOS projects can leverage it without any extra dependency. Here is a description on how this dependency was replaced with the Amplify CLI:

Reviewer: ```suggestion
As of Amplify CLI release ***ADD RELEASE VERSION HERE***, the functionality provided by the `amplify-tools.sh` is integrated with the Amplify CLI. iOS projects can leverage it without adding any extra dependencies or build phases. Here is a description on how this dependency was replaced with the Amplify CLI:
```

> - When running `amplify init --quickstart --frontend ios` like described in our [Getting Started](https://docs.amplify.aws/start/getting-started/setup/q/integration/ios#add-amplify-to-your-application), the relevant files are automatically added to the Xcode project.

Reviewer: ```suggestion
- When running `amplify init --quickstart --frontend ios` as described in the [Getting Started guide](https://docs.amplify.aws/start/getting-started/setup/q/integration/ios#add-amplify-to-your-application), the relevant files are automatically added to the Xcode project.
```

- When running `amplify codegen models`, the `*.swift` files generated under `amplify/generated/models/` are auto-added to the Xcode project in the group called `AmplifyModels`.
**Notes:**

> - The Xcode integration was added to the CLI on version `4.40.0`, so make sure you update your CLI to the latest version.

Reviewer: ```suggestion
- Xcode integration was added to the Amplify CLI on [version `4.40.0`](https://github.com/aws-amplify/amplify-cli/releases/tag/v4.40.0), so make sure you update your CLI to the latest version.
```

> - For existing projects using the build phase script, it can be safely removed.

Reviewer: ```suggestion
- The "***INSERT NAME OF BUILD PHASE HERE***" custom build phase script can be safely removed from existing projects.
```
